Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM node:7
+
+RUN npm install -g \
+      yarn \
+      ember-cli \
+      bower \
+ && apt-get update \
+ && apt-get install -y --no-install-recommends \
+      nginx \
+ && rm -rf /var/lib/apt/lists/* \
+ && rm /var/www/html/index.nginx-debian.html \
+ && ln -sf /dev/stdout /var/log/nginx/access.log \
+ && ln -sf /dev/stderr /var/log/nginx/error.log
```

Comment:
what is the reason to have npm install and apt-get in one command? Cannot they be split into two RUN has they do not depend on each other?

Reply:
The reasoning behind this was having all the installed stuff in one docker layer as per [the docs](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#minimize-the-number-of-layers). I don't think doing this in one `RUN` affects the readability of the Dockerfile.

Reply:
fair point... potentially we need to have a look at this again in timed-backend as well.